devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -884,6 +886,21 @@ bool WholeBodyDynamicsDevice::loadSettingsFromConfig(os::Searchable& config)
     yarp::os::Property prop;
     prop.fromString(config.toString().c_str());
 
+    if (prop.check("devicePeriodInSeconds"))
+    {
+        if(!prop.find("devicePeriodInSeconds").isDouble())
+        {
+            yError() << "wholeBodyDynamics : The devicePeriodInSeconds must be a double";
+            return false;
+        }
+        this->setPeriod(prop.find("devicePeriodInSeconds").asDouble());
+    }
+    else
+    {
+        yWarning() << "wholeBodyDynamics : The devicePeriodInSeconds parameter is not found the default one is used. Period: " << this->getPeriod() << " seconds.";
```

Comment:
```suggestion
        yWarning() << "wholeBodyDynamics : The devicePeriodInSeconds parameter is not found. The default one is used. Period:" << this->getPeriod() << "seconds.";
```

A super small comment. The operator ``<<`` in the ``yarp`` log stream already adds spaces.